src/components/category-box/category-box-component.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { ReactComponent as ArrowExpandIcon } from 'icons/arrow_expand.svg';
+
+import styles from './category-box-styles.module.scss';
+
+const CategoryBox = ({ title, category, isCategoriesBoxesVisible }) => (
+  <div className={cx(
+    styles.box,
+    { [styles.visible]: isCategoriesBoxesVisible }
+  )}>
+    <p className={styles.title}>{title}</p>
+    <div className={styles.categoryContainer}>
+      <p className={styles.category}>{category}</p>
+      <div className={styles.icon}>
+        <ArrowExpandIcon />
```

Comment:
From the designs seems that the arrows only appear on `CategoryBox` hover.
![image](https://user-images.githubusercontent.com/6906348/57760700-66338780-76fc-11e9-8173-7bed73c5690e.png)